packages/gatsby-cli/src/reporter/redux/index.ts

```diff
@@ -31,7 +42,15 @@ const isInternalAction = action => {
   return false
 }
 
-const iface = {
+interface IIface {
+  dispatch(dispatch: any)
+  getStore: () => store
+  onStoreSwap: (fn: storeListener) => void
+  onLogAction: (fn: Dispatch<AnyAction>) => () => void
+  setStore: (s: store) => void
+}
+
+export const iface: IIface = {
```

Comment:
i think this variable was poorly named for `interface` maybe? either way. we should export it with a better name like `ReporterStore`